docs/user_guide/user_guide.md

```diff
@@ -32,13 +32,21 @@ To wire the board to other devices, you can use the pinout diagram or the pin la
 
 The board’s software includes device drivers for onboard devices, a telemetry recorder, and a board application package. Device drivers facilitate interaction with the onboard devices and are utilized by both the telemetry recorder and the board application package. The telemetry recorder, a pivotal tool within the suite, captures data from the devices, offering options to either save this data to an SD card or transmit it through a radio module if one is present on the board. Concurrently, the board application package empowers users to run custom applications on the board. These applications can manipulate the onboard devices and execute custom actions, providing a versatile foundation for developing your own applications.
 
+### 🧩 Board Application Package
```

Comment:
Lets change this to Board Application Tools and put the Telemetry recorder under it and any other packages we create for users to use. Then we can have the other section for Tools as web serial and other mission control readers.